### ISSUE TITLE
Primitive models with long names overlap other info

### ISSUE THREAD

**Issue by hkosova** [swagger-api/swagger-ui#3882]
| Q                               | A
| ------------------------------- | -------
| Bug or feature request?         | Bug
| Which Swagger/OpenAPI version?  | 2.0, 3.0
| Which Swagger-UI version?       | 3.4.3
| How did you install Swagger-UI? | http://petstore.swagger.io
| Which browser & version?        | Chrome 62, Firefox 56
| Which operating system?         | Windows 7

### Demonstration API definition
```yaml
swagger: '2.0'
info:
  version: 0.0.0
  title: text overlapping
paths:
  /:
    get:
      responses:
        200:
          description: OK
          schema:
            $ref: '#/definitions/ObjectModelWithLongLongName'
    
definitions:
  PrimitiveModelWithLongName:
    type: string
    pattern: '^\d+$'

  ObjectModelWithLongLongName:
    type: object
    properties:
      foo:
        $ref: '#/definitions/PrimitiveModelWithLongName'

```
### Expected Behavior
No overlapping text.

### Current Behavior
![overlapped-text](https://user-images.githubusercontent.com/8576823/32619593-4a517018-c58c-11e7-9af5-49c28837f2e3.png)

### Possible Solution
In the CSS for `.swagger-ui .prop-name`, change
```
width: 8em;
```
to
```
min-width: 8em;
```
?

**Comment by webron** [comment]
@Minasokoni @owenconti any of you have time to look at it? seems like a small fix.

**Comment by webron** [comment]
Should be fixed through #3884.
